mathics/eval/files_io/read.py

```diff
@@ -248,12 +248,14 @@ def read_list_from_types(read_types):
     # TODO: look for a better implementation handling "Hold[Expression]".
     #
     read_types = (
-        SymbolHoldExpression
-        if (
-            typ.get_head_name() == "System`Hold"
-            and typ.elements[0].get_name() == "System`Expression"
+        (
+            SymbolHoldExpression
+            if (
+                typ.get_head_name() == "System`Hold"
+                and typ.elements[0].get_name() == "System`Expression"
+            )
+            else typ
```

Comment:
Reformatting may conflict with another PR, but we'll merge this first and I'll deal with the other later.

Reply:
Thanks @mmatera!